src/markdown-pages/install-with-kurl/migrating.md

```diff
@@ -6,31 +6,31 @@ linktitle: "Migrating"
 title: "Migrating to Change kURL Add-Ons"
 ---
 
-Changing the CRI, CSI, or CNI provider on a kURL install is possible by migrating a [Kots](https://kots.io/) application to a new cluster.
+Changing the CRI, CSI, or CNI provider on a kURL install is possible by migrating a [KOTS](https://kots.io/) application to a new cluster.
 
 ## Requirements
 
-* The Kotsadm application must be configured to support [full snapshots](https://kots.io/kotsadm/snapshots/overview/#full-snapshots-recommended).
+* The KOTS application must be configured to support [full snapshots](https://kots.io/kotsadm/snapshots/overview/#full-snapshots-recommended).
 * Velero must be configured to use an external snapshot destination accessible to both the old and new clusters, such as S3.
 * Both the old and new clusters must have the same airgap status. Migrating from airgap to online clusters or vice versa is not supported.
 * The velero add-on version must be at least 1.5.1.
-* The Kotsadm add-on version must be at least 1.45.0 in order for the Kotsadm console certificate to be migrated.
+* The KOTS add-on version must be at least 1.45.0 in order for the KOTS console certificate to be migrated.
```

Comment:
So is KOTS the preferred styling even when we're talking about the add-on, which is called Kotsadm [throughout the docs](https://kurl.sh/docs/add-ons/kotsadm)? If so, should that page change as well? We can't easily change the field name in the installer spec, so that would have to remain kotsadm anyway.

Reply:
Correct, KOTS is the preferred styling when referring to the KOTS product. "kotsadm" is legacy and not preferred branding, though we may still need to call it that when we're directly referencing code where it may still be called kotsadm. 

That page you linked likely should be updated, at least in the title. Notice we call it "KOTS" vs "kotsadmin" on the prior page linking there (https://kurl.sh/add-ons).

Reply:
Got it. Then I will approve this and open a new PR to update that page and any others I see.

Reply:
Cool, thanks @ajp-io !